pricegraph/src/orderbook.rs

```diff
@@ -49,11 +54,19 @@ impl Orderbook {
         let mut orders = OrderCollector::default();
         let mut users = UserMap::default();
 
+        const MIN_AMOUNT_U128: u128 = MIN_AMOUNT as _;
+        const MIN_AMOUNT_U256: U256 = U256([MIN_AMOUNT as _, 0, 0, 0]);
+
         for element in elements {
+            if element.remaining_sell_amount < MIN_AMOUNT_U128 || element.balance < MIN_AMOUNT_U256
+            {
+                continue;
+            }
+
```

Comment:
Would it be appropriate to have function declared on "element" called "is_infinitesimal" or "is_negligible" where this logic lives? Then we could `elements.filter(|e| !infinitesimal(e))` rather than `continue`.